### PR TITLE
Added Documentation support for Bazel installation in Linux distros

### LINKED ISSUE #3439: Bazel installation for Debian Does not work
### Description of the bug:

'Sudo apt install bazel' command does not work in debian based distros. The command is outdated.

### Link to documentation:

https://github.com/carbon-language/carbon-lang/blob/trunk/docs/project/contribution_tools.md

### What should it say instead?

It must have a link to the bazel-[version]-linuxx86_64.sh file and the repo that contains the assets and the installation steps after that. 

### Any other information, logs, or outputs that you want to share?

_No response_

### PR DESCRIPTION
**Pull request by Dananjay996**
This Pull Request added support for Bazel installation and setup in the contribution_tools.md file un the docs directory. It added a link to the releases page of the bazel repository for debian/ubuntu users to download the binary and execute the commands provided in the documentation. 

Closes #3439

### CODE REVIEW

**Comment by jonmeow** [comment]
Sorry, I'm trying to understand here, which distribution are you using? It sounds like you're not using Debian, and I think these instructions may be more specific. I'm hesitant to get into instructions for other distributions: typically I would expect to just say to refer to, for example, Bazel's install instructions.

**Comment by jonmeow** [comment]
By the way, here is the Debian bazel package: https://qa.debian.org/developer.php?email=team%2Bbazel%40tracker.debian.org -- maybe it's just `bazel-bootstrap` on debian now?

**Comment by Dananjay996** [comment]
> Sorry, I'm trying to understand here, which distribution are you using? It sounds like you're not using Debian, and I think these instructions may be more specific. I'm hesitant to get into instructions for other distributions: typically I would expect to just say to refer to, for example, Bazel's install instructions.

I am using Ubuntu and this installation method is for installing Bazel on Ubuntu.

**Comment by jonmeow** [comment]
> I am using Ubuntu and this installation method is for installing Bazel on Ubuntu.

Thanks!  Here's information on the Ubuntu package information:

https://bazel.build/install/ubuntu#install-on-ubuntu

I think at most we want to refer to https://bazel.build/install, since it's authoritative.

**Comment by Dananjay996** [comment]
> By the way, here is the Debian bazel package: https://qa.debian.org/developer.php?email=team%2Bbazel%40tracker.debian.org -- maybe it's just `bazel-bootstrap` on debian now?

I think my issue is Ubuntu Specific. I should have not mentioned Debian in the PR. My bad. Should I update the PR and remove debian? I added this because I faced this issue while setting up for myself and I didn't want others to experience the same.

File: docs/project/contribution_tools.md [hunk 1]
```diff
@@ -68,6 +68,23 @@ bazel test //...:all
...
+> # Install bazel.
+> chmod +x bazel-<latest_stable_release_version>-installer-linux-x86_64.sh
+> ./bazel-<latest_stable_release_version>-installer-linux-x86_64.sh --user
+> export PATH="$PATH:$HOME/bin"
+> ```
+>
+> Go to [bazel release page](https://github.com/bazelbuild/bazel/releases) and
+> download the binary corresponding to your system. For example, if you are
+> using Linux, download the binary named
+> 'bazel-<latest_stable_release_version>-installer-linux-x86_64.sh'
```
Comment (jonmeow) [inline docs/project/contribution_tools.md]:
```suggestion
> NOTE: If you don't have a `bazel` package, see [Bazel's install instructions](https://bazel.build/install) for help.
```

**Comment by Dananjay996** [comment]
> > I am using Ubuntu and this installation method is for installing Bazel on Ubuntu.
> 
> Thanks!  Here's information on the Ubuntu package information:
> 
> https://bazel.build/install/ubuntu#install-on-ubuntu
> 
> I think at most we want to refer to https://bazel.build/install, since it's authoritative.

Cool, l'll update.

**Review by jonmeow — APPROVED** [review]

**Pull request merged**